src/desktop/apps/article/components/NewsArticle.tsx

```diff
@@ -4,13 +4,16 @@ import Waypoint from 'react-waypoint'
 
 interface Props {
   article: any
+  display?: any
   isActive: boolean
   isFirstArticle: boolean
   isMobile: boolean
   isTruncated: boolean
   nextArticle: any
   onActiveArticleChange: (id: string) => void
   onDateChange: (date: string) => void
+  relatedArticlesForCanvas?: any
+  renderTime?: any
```

Comment:
Let's use actual types in our interfaces whenever possible, not obvious if `renderTime` is a `number` or `boolean`.